uat/testing-features/src/main/resources/greengrass/components/recipes/client_ipc.yaml

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
just to keep in mind, the iot core ipc stuff should live in OTF repo.  once we have it fully fleshed out here and have assertion server we should think about moving all of it over there